pyproject.toml

```
@@ -7,7 +7,7 @@ authors = [
 ]
 dependencies = [
     "packaging",
-    "astropy>=4.3",
+    "astropy>=5.3.2",
```

astropy 5.x is no longer supported, might as well bump to 6.x ?
```suggestion
    "astropy>=6",
```